features/smoke-tests/supplier/opportunities.feature

```diff
@@ -34,46 +33,115 @@ Scenario Outline: User can filter by individual status
   Given I am on the /digital-outcomes-and-specialists/opportunities page
   When I note the result_count
   And I check '<status>' checkbox
-  And I click the 'Filter' button
+  And I wait for the page to reload
   Then I see that the stated number of results does not exceed that result_count
   And I see all the opportunities on the page are of the '<status>' status
 
   Examples:
-    | status |
-    | Open   |
-    | Closed |
+    | status       |
+    | Open         |
+    | Closed       |
+    | Awarded      |
```

Comment:
These are going to need to be removed. Just Open and Closed now.

Reply:
- [ ] Sigh.